src/plugins/controls/public/options_list/components/options_list_popover.tsx

```diff
@@ -21,20 +23,18 @@ import {
   EuiIcon,
   EuiTitle,
 } from '@elastic/eui';
-import { isEmpty } from 'lodash';
-
 import { useReduxEmbeddableContext } from '@kbn/presentation-util-plugin/public';
-import { optionsListReducers } from './options_list_reducers';
+
+import { optionsListReducers } from '../options_list_reducers';
 import { OptionsListStrings } from './options_list_strings';
-import { OptionsListReduxState } from './types';
+import { OptionsListReduxState } from '../types';
 
-export const OptionsListPopover = ({
-  width,
-  updateSearchString,
-}: {
+export interface OptionsListPopoverProps {
   width: number;
   updateSearchString: (newSearchString: string) => void;
-}) => {
+}
+
+export const OptionsListPopover = ({ width, updateSearchString }: OptionsListPopoverProps) => {
```

Comment:
I believe most of the controls-related components follow the style **without** the `FC<Props>` so I think I'll stick with that? Honestly not a huge deal, but if we wanted to be **that** thorough about code consistency, it honestly would be easier to remove `FC<Props>` from everywhere because, at least in `main` right now, only time slider uses that syntax 😆